Testnet/DAOContract/DAOContract/DAOContract.cs

```diff
@@ -0,0 +1,277 @@
+﻿using Stratis.SmartContracts;
+using System;
+
+[Deploy]
+public class DAOContract : SmartContract
+{
+    public Address Owner
+    {
+        get => State.GetAddress(nameof(Owner));
+        private set => State.SetAddress(nameof(Owner), value);
+    }
+
+    public uint MinQuorum => WhitelistedCount / 2 + 1;
```

Comment:
maybe just to be safe...`WhitelistedCount > 2 ? WhitelistCount / 2 + 1 : WhitelistedCount`

Reply:
If count is 0 then MinQuorum is 1. This is fine because we dont want that proposal can be executed without any vote. 
if count is 1 then MinQuorum is 1 . This is fine as well
if count is 2 then MinQuorum is 2 . This is acceptable that we dont want that single person can manipulate the system.

Reply:
0 / 2 will throw.